cpp/include/cugraph/algorithms.hpp

```diff
@@ -760,7 +760,8 @@ std::pair<std::unique_ptr<Dendrogram<vertex_t>>, weight_t> leiden(
   graph_view_t<vertex_t, edge_t, false, multi_gpu> const& graph_view,
   std::optional<edge_property_view_t<edge_t, weight_t const*>> edge_weight_view,
   size_t max_level    = 100,
-  weight_t resolution = weight_t{1});
+  weight_t resolution = weight_t{1},
+  weight_t theta      = weight_t{1});
```

Comment:
Documentation missing for theta.

Reply:
Not really a comment for this PR but `rng_state` appear right after handle in `sample_neighbors_adjacency_list` and `sample_neighbors_edgelist` but appears right before the last 3 Boolean input parameters in `uniform_neighbor_sample`.

@ChuckHastings This can be confusing and we need some consistency in input parameter ordering.

I think `rng_state` appearing after graph data makes little more logical sense (graph data is more essential than rng_state in computing Leiden).